pkg/controllers/apply_controller.go

```diff
@@ -148,13 +148,9 @@ func (r *ApplyWorkReconciler) applyManifests(manifests []workv1alpha1.Manifest,
 		} else {
 			var obj *unstructured.Unstructured
 			result.identifier = buildResourceIdentifier(index, rawObj, gvr)
-			if !findWorkReference(rawObj.GetOwnerReferences(), owner) {
-				klog.V(5).InfoS("manifest is not owned by the work-api, will not be applied.", "gvr", gvr, "obj", rawObj.GetName())
-				return results
-			}
 			rawObj.SetOwnerReferences(insertOwnerReference(rawObj.GetOwnerReferences(), owner))
```

Comment:
since we pass the owner in the applyUnstructured function, we can set that inside the function.

Reply:
I have removed the owner from the parameter for applyUnstructured.
The applyUnstructured will now use rawObj's owner reference.